src/components/Footer.tsx

```diff
@@ -49,6 +57,14 @@ const SocialImg = styled.img`
   margin: 8px;
 `
 
+const SocialLinks = styled.div`
+    display: flex;
+    height: 40px;
+    margin: 10px 0;
+    font-size: 10px;
+    color: colors.fontColor1;
```

Comment:
Oops, this should probably be a template variable:

```suggestion
    color: ${colors.fontColor1};
```